.github/renovate.json

```diff
@@ -5,7 +5,8 @@
     ":rebaseStalePrs",
     "group:linters",
     "group:test",
-    ":preserveSemverRanges"
+    ":preserveSemverRanges",
+    "helpers:pinGitHubActionDigests"
```

Comment:
Upon further investigation it's actually doing already this but we are hitting the limits of the current default which is why this wasn't being done automatically. If you can revert this change that would be great 👍